cgmanifest.json

```diff
@@ -210,6 +210,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "azure-storage-cpp",
+          "version": "2.7.3",
+          "downloadUrl": "https://github.com/Azure/azure-storage-cpp/archive/v2.7.3.tar.gz"
```

Comment:
The version seems incorrect and the link returns a 404. It probably should be this one: [https://github.com/Azure/azure-storage-cpp/archive/v7.3.0.tar.gz](https://github.com/Azure/azure-storage-cpp/archive/v7.3.0.tar.gz).